velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -3836,3 +3836,18 @@ TEST_F(DateTimeFunctionsTest, fromUnixtimeDouble) {
   });
   assertEqualVectors(expected, actual);
 }
+
+TEST_F(DateTimeFunctionsTest, toISO8601TestDate) {
+  const auto toISO8601 = [&](std::optional<int32_t> date) {
+    return evaluateOnce<std::string, int32_t>(
+        "to_iso8601(c0)", {date}, {DATE()});
+  };
+
+  EXPECT_EQ("1970-01-01", toISO8601(DATE()->toDays("1970-01-01")));
```

Comment:
for readability, consider passing date string to the lambda:

```
EXPECT_EQ("1970-01-01", toIso8601("1970-01-01"));
```

Reply:
Good point, done.